deleting_a_router.md

```diff
@@ -1,10 +1,10 @@
-### Deleting a Network Router
+
+# Deleting a Network Router
```

Comment:
```suggestion
# Deleting a Network Router
```

Reply:
I think this change was pulled out into a separate PR.

Reply:
Ah, I didn't see the merge conflict